main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
Hi @Tobias-Fischer,
The purpose of `zOffset` is the `table height` for the case the system cannot get that information from the `ARE subsystem`, and it can be set/changed easily by users due to the different setup at different lab. That is why I introduce it in `ini` file, rather than hard-coding in `subSystem_Karma`. The name should be `zTable` or similar to better transfer the idea. 
Please tell me your idea!